controllers/tasksRequests.js

```diff
@@ -167,6 +168,8 @@ const approveTaskRequest = async (req, res) => {
     if (response.isTaskRequestInvalid) {
       return res.boom.badRequest("Task request was previously approved or rejected.");
     }
+    await updateUserStatusOnTaskUpdate(user.username);
```

Comment:
Is this function update user-status to active??

Reply:
yes